backend/app/converters/lib/marcxml_base_map.rb

```diff
@@ -225,42 +223,6 @@ def corp_template
           :obj => :name_corporate_entity,
           :rel => :names,
           :map => name_corp_map,
-        },
-        "//datafield[@tag='410']" => {
-          :obj => :name_corporate_entity,
-          :rel => :names,
-          :map => name_corp_map,
-          :defaults => {
-            :name_order => 'direct',
-            :source => 'ingest'
-          }
-        },
-        "//datafield[@tag='411']" => {
-          :obj => :name_corporate_entity,
-          :rel => :names,
-          :map => name_corp_map,
-          :defaults => {
-            :name_order => 'direct',
-            :source => 'ingest'
-          }
-        },
-        "//datafield[@tag='610']" => {
-          :obj => :name_corporate_entity,
-          :rel => :names,
-          :map => name_corp_map,
-          :defaults => {
-            :name_order => 'direct',
-            :source => 'ingest'
-          }
-        },
```

Comment:
Yeah, I think there's a good chance that the 410, 411, 610, 611 tags all have the same bug, so maybe it's worth commenting on the JIRA to ask whether anyone can confirm this.  I wonder whether 410 and 4100 should be moved into the BASE_RECORD_MAP so they get treated in the same way as 610 and 611...